perception/navigator_vision/datasets/project_bags.yaml

```diff
@@ -213,3 +213,571 @@ datasets:
         start: 0
         stop: 9
 
+  -
+      name: navigator_wahlberg_pg_2018-02-04
```

Comment:
Please follow convention from first dataset of `pointgrey` rather than `pg`, so we can later segment it by camera if needed